AUTHORS

```diff
@@ -3021,5 +3021,106 @@ David Gilbertson <gilbertson.david@gmail.com>
 Sergey Zelenov <sergey.zelenov@getyourguide.com>
 Eric Bickle <ebickle@users.noreply.github.com>
 Joe Pea <joe@trusktr.io>
+ExE Boss <3889017+ExE-Boss@users.noreply.github.com>
+Mateusz Krawczuk <krawczukmat@gmail.com>
+Jonathan MERCIER <jmercier@cng.fr>
+Ujjwal Sharma <ryzokuken@igalia.com>
```

Comment:
This is definitely a duplicate. Did you check if there are more?

Reply:
@lpinca  This email is also not present in `.mailmap` should I update it in the `.mailmap` as well?

Reply:
Yes, add it here https://github.com/nodejs/node/blob/dd0c5228acaf5695f96809e50771ead3167b0f44/.mailmap#L414.

Reply:
Cool 👍

Reply:
I think the `update-authors` also need to change for this.